org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/model/ModelViewer.java

```diff
@@ -374,6 +393,55 @@ public void refresh() {
 		viewer.refresh();
 	}
 	
+	@Override
+	public void updateModel(T model) {
+		T old      = this.model;
+		this.model = model;
+		validate();
+		setSelection(new StructuredSelection(model));
+		setSeriesItem(old, model);
+	}
+	
+	private void setSeriesItem(T old, T model) {
+		
+		// Find a view reference claiming to edit the thing which we changed.
+		IViewReference ref = Arrays.stream(PageUtil.getPage(site).getViewReferences())
+				                      .filter(des->isScanView(des))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)